wcivf/apps/people/managers.py

```diff
@@ -174,9 +174,12 @@ def update_or_create_from_ynr(self, person):
     def get_by_pk_or_redirect_from_ynr(self, pk):
         try:
             return self.get(pk=pk)
-        except self.model.DoesNotExist:
+        except self.model.DoesNotExist as initial_e:
             from people.models import PersonRedirect
 
-            return self.get(
-                PersonRedirect.objects.get(old_person_id=pk).new_person_id
-            )
+            try:
+                return self.get(
+                    PersonRedirect.objects.get(old_person_id=pk).new_person_id
+                )
```

Comment:
this was wrong in the original PR too, but this needs to be

```
return self.get(pk=PersonRedirect.objects.get(old_person_id=pk).new_person_id)
```

Just calling `.get(1)` rather than `.get(pk=1)` will throw `TypeError: cannot unpack non-iterable int object`